packages/plugin/src/generators/create-package/files/e2e/__cliName__.spec.ts__tmpl__

```diff
@@ -1,31 +1,56 @@
-import {
-  checkFilesExist,
-  removeTmpProject,
-  tmpFolder,
-  uniq,
-} from '@nx/plugin/testing';
 import { execSync } from 'child_process';
-import { join } from 'path';
-
-describe('<%= cliName %> e2e', () => {
-  let project: string;
-  beforeAll(async () => {
-    // create a workspace with cli
-    project = uniq('<%= cliName %>');
-    execSync(`npx <%= cliName %>@1.0.0 ${project}`, {
-      cwd: tmpFolder(),
-      env: process.env,
-    });
-  }, 240_000);
+import { join, dirname } from 'path';
+import { mkdirSync, rmSync } from 'fs';
+
+describe('<%= cliName %>', () => {
+  let projectDirectory: string;
 
   afterAll(() => {
-    // Remove the generated project from the file system
-    removeTmpProject(project);
+    // Cleanup the test project
+    rmSync(projectDirectory, {
+      recursive: true,
+      force: true,
+    });
   });
 
-  it('should create project using <%= cliName %>', () => {
-    expect(() =>
-      checkFilesExist(join(tmpFolder(), project, 'package.json'))
-    ).not.toThrow();
+
+  it('should be installed', () => {
+    projectDirectory = createTestProject();
+
+    // npm ls will fail if the package is not installed properly
+    execSync('<%= packageManagerCommands.list %> <%= pluginPackageName %>', {
+      cwd: projectDirectory,
+      stdio: 'inherit',
+    });
   });
 });
+
+/**
+ * Creates a test project with create-nx-workspace and installs the plugin
+ * @returns The directory where the test project was created
+ */
+function createTestProject(extraArgs: string = '') {
+  const projectName = 'test-project';
+  const projectDirectory = join(process.cwd(), 'tmp', projectName);
+
+  // Ensure projectDirectory is empty
+  rmSync(projectDirectory, {
+    recursive: true,
+    force: true,
+  });
+  mkdirSync(dirname(projectDirectory), {
+    recursive: true,
+  });
+
+  execSync(
+    `<%= packageManagerCommands.exec %> --yes <%= cliName %>@e2e ${projectName} ${args}`,
+    {
+      cwd: dirname(projectDirectory),
+      stdio: 'inherit',
+      env: process.env,
```

Comment:
Note: The default is `process.env`, so it's not necessary unless we're adding more values to it.

Reply:
Let me try without it in another PR.

I know it shouldn't make any difference... but somehow it does sometimes?